docs/en/latest/admin-api.md

```diff
@@ -1684,9 +1690,12 @@ Proto resource request address: /apisix/admin/protos/{id}
 
 ### Request Body Parameters
 
-| Parameter | Required | Type    | Description                          | Example                       |
-|-----------|----------|---------|--------------------------------------| ----------------------------- |
-| content   | True     | String  | content of `.proto` or `.pb` files   | See [here](./plugins/grpc-transcode.md#enabling-the-plugin)         |
+| Parameter | Required | Type      | Description                          | Example                       |
+|-----------|----------|-----------|--------------------------------------| ----------------------------- |
+| content   | True     | String    | content of `.proto` or `.pb` files   | See [here](./plugins/grpc-transcode.md#enabling-the-plugin)         |
```

Comment:
need change zh doc

Reply:
The Chinese documentation does not contain a `Proto` paragraph, so I will not add it. In fact, I didn't find a Proto attribute paragraph in any of the Chinese documentation. Adding a non-existent paragraph as a whole is not within the scope of this PR, and it won't be done here.